packages/core/js-client/src/clientPeer/types.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import type { Node } from "./commonTypes.js";
+import type { Node } from "@fluencelabs/interfaces";
```

Comment:
Does that refer to a published library, or to a local path?

If it refers to a published library, you might have bad time updating these types.

Reply:
Moved it to js-client side.